images/custom-scorecard-tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the custom-scorecard-tests binary
+FROM --platform=x86_64 golang:1.17.9 as builder
+ARG TARGETARCH
```

Comment:
Cert Manager Operator is x86 ATM. @s-urbaniak Do we want to add more architectures?